packages/engine-core/src/library/LibraryEngine.ts

```diff
@@ -98,7 +98,7 @@ export class LibraryEngine extends Engine {
     this.datasourceOverrides = config.datasources
       ? this.convertDatasources(config.datasources)
       : {}
-    if (config.enableEngineDebugMode) {
+    if (config.enableDebugLogs) {
```

Comment:
This change is also available standalone at https://github.com/prisma/prisma/pull/9030